pingtest.sh

```diff
@@ -1,18 +1,31 @@
 #! /bin/bash
 
+<<<<<<< HEAD
+# Argument 1 ($1) is the server's url, and Argument 2 ($2)is the email address to which an error message should be sent
```

Comment:
it's worth looking up docstring conventions - you don't have to follow it exactly but consider something like:

```
# Example Use:
# ./pingtest.sh url email@address.com
#
# Arguments:
# $1: string: server's url
# $2: string: email address to which any error messages will be sent